app/formations/[id]/opengraph-image.tsx

```diff
@@ -87,6 +104,23 @@ function ServerFormation({ formation, characters, map, artifact }: any) {
         />
         {formation.username}
       </div>
+      <div
+        style={{
+          display: "flex",
+          flexDirection: "column",
+          position: "absolute",
+          right: "40px",
+          justifyContent: "center",
+          alignItems: "flex-end",
+          fontSize: "24px",
+          fontWeight: "bold",
+          height: "100%",
+        }}
+      >
+        {formation.tags.map((x: string) => (
+          <p key={x}>{x}</p>
+        ))}
+      </div>
```

Comment:
These feel a little stranded right now- perhaps make them look like the website tags and experiment putting them above the username/image in the bottom left so they have a bit of cohesion